incidenter/program.py

```diff
@@ -0,0 +1,145 @@
+from os import getenv
+import re
+
+from autokitteh import next_event, subscribe
+from autokitteh.slack import slack_client
+
+from height import add_task_message
+from height import create_task
+from zoom import create_meeting
+
+
+CHANNEL_PREFIX = getenv("CHANNEL_PREFIX")
+
+slack_client = slack_client("slack")
+
+pattern = re.compile(r"<.+> incident *(.*)")
+
+
+def on_slack_app_mention(event):
+    m = pattern.match(event.data.text)
+    if not m:
+        print("irrelevant")
+        return
+
+    incident = _start(event.data.user, event.data.channel, event.data.ts, m.groups()[0])
+
+    _track(incident)
+
+
+def _start(
+    slack_user: str, trigger_channel_id: str, trigger_ts: str, title: str
+) -> dict:
+    task = create_task(title, "New incident created from slack", "inProgress")
+
+    zoom_url = create_meeting(f"Incident: {title}")
+
+    channel = slack_client.conversations_create(
+        name=f"{CHANNEL_PREFIX}{task["index"]}", is_private=False
+    ).get("channel")
+
+    slack_client.conversations_setTopic(
+        channel=channel["id"],
+        topic=f"⚠ Incident: {title}",
```

Comment:
Optional nit (not relevant in line 59 because it would be cumbersome there)
```suggestion
        topic=f":warning: Incident: {title}",
```